README.md

```diff
@@ -9,3 +9,23 @@ A good overview of all its tickets and their status can be found here: https://g
 Read more about the Badger DAO and its community at https://badger.com/dao-and-community.
 
 ![f](https://badger.com/images/badger-logo.svg)
+
+
+# Multisig Adresses
+
+| Label | Description | Address (Links) |
+|-|-|-|
+|`dev`|Governance/admin rights; set parameters on vaults and strategies, queue/execute timelock txs, etc.|Mainnet: `0xB65cef03b9B89f99517643226d76e286ee999e77` ([Etherscan](https://etherscan.io/token/0xB65cef03b9B89f99517643226d76e286ee999e77), [Gnosis Safe](https://gnosis-safe.io/app/eth:0xB65cef03b9B89f99517643226d76e286ee999e77/), [Zapper](https://zapper.fi/account/0xB65cef03b9B89f99517643226d76e286ee999e77))|
```

Comment:
Etherscan link is wrong, should be: `https://etherscan.io/address/0xB65cef03b9B89f99517643226d76e286ee999e77`